docs/en/revocation-lists.rst

```diff
@@ -212,142 +231,220 @@ The following HTTP Status Codes MUST be supported:
 
-  - *error*. The error code, as registerd in the table below.
+.. code::
+
+  {
+    "alg": "ES256",
+    "typ": "revocation-error+jwt",
+    "kid": "Issuer-JWK-KID"
+  }
+ .
+  {
+    "iss": "https://issuer.example.org",
+    "jti": "6f204f7e-e453-4dfd-814e-9d155319408c"
+    "credential_hash": $CREDENTIAL-HASH,
+    "credential_hash_alg": "sha-256",
+    "credential_status_validity": false,
+    "credential_status": {
+      "revoked": true,
+      "suspended": false
+    },
+    "cnf": {
+      "jwk": {
+        "alg": "ES256",
+        "kty": "EC",
+        "crv": "P-256",
+        "x": "_2ySUmWFjwmraNlo15r6dIBXerVdy_NpJuwAKJMFdoc",
+        "y": "MV3C88MhhEMba6oyMBWuGeB3dKHP4YADJmGyJwwILsk"
+      }
+    }
+  }
+
+The Revocation Assertion Error object MUST contain the following parameters:
+
+  - *error*. The error code, as registerd in the table below;
   - *error_description*. Text in human-readable form providing further details to clarify the nature of the error encountered.
 
-Error codes are meant to provide additional information about the failure so that the User can be informed and take the appropriate action.
-The following Error Codes MUST be supported:
+Errors are meant to provide additional information about the failure so that the User can be informed and take the appropriate action.
+The `error` parameter for the Revocation Assertion Error object MUST be set with one of the values defined in the table below, in addition to the values specified in :rfc:`6749#section-5.2`:
 
-.. list-table:: 
+Here's the fixed table in reStructuredText (reST) format:
+
+.. list-table::
     :widths: 20 80
     :header-rows: 1
 
     * - **Error Code**
       - **Description**
     * - ``invalid_request``
       - The request is not valid due to the lack or incorrectness of one or more parameters. (:rfc:`6749#section-5.2`).
-
-
-Below a non-normative example of an HTTP Response with an error.
+    * - ``credential_already_revoked``
+      - The Digital Credential is already revoked.
+    * - ``credential_updated``
+      - One or more information contained in the Digital Credential are changed. The `error_description` field SHOULD contain a human-readable text describing the general parameters updated without specifying each one.
+    * - ``credential_invalid``
+      - The Digital Credential is invalid. The `error_description` field SHOULD contain the reason of invalidation.
+    * - ``invalid_request_signature``
+      - The Revocation Assertion Request signature validation has failed. This error type is used when the proof of possession of the Digital Credential is found not valid within the Revocation Assertion Request.
+    * - ``credential_not_found``
+      - The `credential_hash` value provided in the Revocation Assertion Request doesn't match with any active Digital Credential.
+    * - ``unsupported_hash_alg``
+      - The hash algorithm set in `credential_hash_alg` is not supported.
+
```

Comment:
It seems we already have a table with HTTP status code at line 225.
What is the meaning of these?

Reply:
> It seems we already have a table with HTTP status code at line 225. What is the meaning of these?

These status codes refer to the revocation assertion flow. The other table shows status assertion codes.